hat/assets/js/apps/Iaso/domains/app/translations/fr.json

```
@@ -22,6 +22,10 @@
     "blsq.treeview.search.results.label.display": "Afficher",
     "blsq.treeview.search.results.label.resultsLower": "résultat(s)",
     "iaso.algo.addRun": "Exécuter un algorithme",
+    "iaso.analytics.consent": "Consentement à l'analyse",
+    "iaso.analytics.consent.accept": "Accepter",
+    "iaso.analytics.consent.decline": "Refuser",
+    "iaso.analytics.consent.description": "Nous utilisons Sentry pour collecter des données de rejoue de session afin d'améliorer la qualité de nos services et de débugger les problèmes. Cela inclut la capture de vos interactions avec notre application.",
```

```suggestion
    "iaso.analytics.consent.description": "Nous utilisons Sentry pour collecter des données de rejeu de session afin d'améliorer la qualité de nos services et de débugger les problèmes. Cela inclut la capture de vos interactions avec notre application.",
```

Sounds a little better to me 😄